tests/wpt/mozilla/tests/mozilla/bluetooth/idl-Bluetooth.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+'use strict';
+test(() => {
+    //assert_throws('ReferenceError', function () { new Bluetooth() });
+    //assert_throws(null, function () { Bluetooth() });
+    assert_true('requestDevice' in window.navigator.bluetooth);
+}, 'Bluetooth IDL test');
```

Comment:
To be fair, if our WebIDL files differ from the spec, having idlharness tests would allow us to verify that our changes to match the spec actually do so ;)